llvmCoverageMappingC/src/main/cpp/CoverageMappingC.cpp

```diff
@@ -241,7 +241,13 @@ void LLVMKotlinInitializeTargets() {
     INIT_LLVM_TARGET(X86)
     INIT_LLVM_TARGET(WebAssembly)
 #elif KONAN_LINUX
+    INIT_LLVM_TARGET(AArch64)
+    INIT_LLVM_TARGET(ARM)
+    INIT_LLVM_TARGET(Mips)
+    INIT_LLVM_TARGET(X86)
 #elif KONAN_WINDOWS
+    INIT_LLVM_TARGET(ARM)
```

Comment:
Why no AArch64 here? We support linux_arm64 on Windows hosts.

Reply:
It's the same issue as with the unsupported `android_arm64`. Looks like we need to rebuild LLVM toolchain for Windows.

Reply:
Hmm, but now linux_arm64 works.

Reply:
```
msys2-mingw-w64-x86_64-gcc-7.3.0-clang-llvm-lld-6.0.1\bin> ./llvm-config --libs aarch64
llvm-config: unknown component name: aarch64
```
So we need to rebuild the toochain  to use LLVM API for AArch64 target.

Reply:
```
msys2-mingw-w64-x86_64-gcc-7.3.0-clang-llvm-lld-6.0.1\bin> llc --version
LLVM (http://llvm.org/):
  LLVM version 6.0.1
  Optimized build.
  Default target: x86_64-w64-windows-gnu
  Host CPU: ivybridge

  Registered Targets:
    aarch64    - AArch64 (little endian)
    aarch64_be - AArch64 (big endian)
    amdgcn     - AMD GCN GPUs
    arm        - ARM
    arm64      - ARM64 (little endian)
    armeb      - ARM (big endian)
    bpf        - BPF (host endian)
    bpfeb      - BPF (big endian)
    bpfel      - BPF (little endian)
    hexagon    - Hexagon
    lanai      - Lanai
    mips       - Mips
    mips64     - Mips64 [experimental]
    mips64el   - Mips64el [experimental]
    mipsel     - Mipsel
    msp430     - MSP430 [experimental]
    nvptx      - NVIDIA PTX 32-bit
    nvptx64    - NVIDIA PTX 64-bit
    ppc32      - PowerPC 32
    ppc64      - PowerPC 64
    ppc64le    - PowerPC 64 LE
    r600       - AMD GPUs HD2XXX-HD6XXX
    sparc      - Sparc
    sparcel    - Sparc LE
    sparcv9    - Sparc V9
    systemz    - SystemZ
    thumb      - Thumb
    thumbeb    - Thumb (big endian)
    wasm32     - WebAssembly 32-bit
    wasm64     - WebAssembly 64-bit
    x86        - 32-bit X86: Pentium-Pro and above
    x86-64     - 64-bit X86: EM64T and AMD64
    xcore      - XCore
```

Reply:
And current clang (8.0.0):
```
F:\msys64\mingw64\bin>clang --version
clang version 8.0.0 (tags/RELEASE_800/final)
Target: x86_64-w64-windows-gnu
Thread model: posix
InstalledDir: F:\msys64\mingw64\bin

F:\msys64\mingw64\bin>llvm-config --libs aarch64
-lLLVMAArch64Disassembler -lLLVMMCDisassembler -lLLVMAArch64CodeGen -lLLVMGlobalISel -lLLVMSelectionDAG -lLLVMAsmPrinter -lLLVMCodeGen -lLLVMTarget -lLLVMScalarOpts -lLLVMInstCombine -lLLVMAggressiveInstCombine -lLLVMTransformUtils -lLLVMBitWriter -lLLVMAnalysis -lLLVMProfileData -lLLVMObject -lLLVMBitReader -lLLVMCore -lLLVMAArch64AsmParser -lLLVMMCParser -lLLVMAArch64Desc -lLLVMAArch64Info -lLLVMAArch64AsmPrinter -lLLVMMC -lLLVMDebugInfoCodeView -lLLVMDebugInfoMSF -lLLVMBinaryFormat -lLLVMAArch64Utils -lLLVMSupport -lLLVMDemangle
```